### PR TITLE
enhance/navigation

### PR DESCRIPTION
**Pull request by haritonasty**
## Changes

upd main nav

## Checklist

- [x] I've performed a self-review, followed all rules from [Frontend style guide](https://www.notion.so/santiment/Front-end-style-guide-81750096b38c4bea9a29b14fd4ab8667)
- [x] If I make changes to another person's module, I've asked how to use it or request a review
- [x] I've updated the [documentation](https://github.com/santiment/academy), if necessary (Keyboard shortcuts, Account settings)
- [x] I've checked night mode, mobile & tablet screens (if have changes in UI)
- [x] I've added tests (if necessary)

## Screenshots or GIFs

image

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -39,13 +39,17 @@
...
     </style>
     <div id="root">
       <div class="loader">
-        <img width="128" height="128" src="%PUBLIC_URL%/logo.svg" alt="Sanbase logo" />
+        <svg width="128" height="128" viewBox="0 0 32 32" fill="none">
+          <path fill="#fff" d="M32 16a16 16 0 11-32 0 16 16 0 0132 0z"/>
+          <path fill="#181B2B" d="M8.23 16a1.7 1.7 0 01-1.71 1.69A1.7 1.7 0 014.8 16c0-.93.77-1.69 1.72-1.69.94 0 1.71.76 1.71 1.69zM25.48 17.69A1.7 1.7 0 0027.2 16a1.7 1.7 0 00-1.72-1.69A1.7 1.7 0 0023.77 16c0 .93.77 1.69 1.71 1.69zM19.43 9.61a6.5 6.5 0 00-4.4-.67c-1.38.32-2.57 1.26-2.8 2.84-.18 1.3.22 2.33.9 3.14a9.31 9.31 0 002.27 1.77c.81.5 1.48.9 1.97 1.38.46.44.7.9.7 1.5 0 .69-.2 1.04-.43 1.24-.24.22-.63.38-1.19.42a5.43 5.43 0 01-3.3-.97l-1.21 1.56a7.42 7.42 0 004.65 1.37 3.92 3.92 0 002.4-.94 3.48 3.48 0 001.07-2.68c0-1.26-.55-2.19-1.28-2.9-.7-.67-1.6-1.2-2.32-1.65a7.52 7.52 0 01-1.8-1.36c-.35-.42-.55-.9-.45-1.61.08-.59.5-1.02 1.28-1.2a4.5 4.5 0 013 .5l.94-1.74z"/>
```
Comment (GarageInc) [inline public/index.html:50]: Colors? (dark mode)

File: src/components/Navbar/NavbarMore/NavbarMore.js [hunk 2]
```diff
@@ -1,59 +1,80 @@
...
-import LinkWithArrow from '../Link'
-import externalStyles from '../Watchlists/MarketDropdown.module.scss'
+import SantimentLogo from '../SantimentProductsTooltip/SantimentLogo'
+import {
+  BUSINESS_PRODUCTS,
+  CHAIN_PRODUCTS
+} from '../SantimentProductsTooltip/Products'
 import styles from './NavbarMore.module.scss'
+import ContactUs from '../../ContactUs/ContactUs'
+import cx from 'classnames'
```
Comment (GarageInc) [inline src/components/Navbar/NavbarMore/NavbarMore.js:11]: Order?

File: src/components/Navbar/NavbarMore/NavbarMore.module.scss [hunk 3]
```diff
@@ -1,31 +1,129 @@
...
+}
+
+.imgWrapper {
+  width: 40px;
+  height: 40px;
+  display: flex;
+  margin-right: 12px;
+  align-items: center;
+  justify-content: center;
+  border-radius: 4px;
```
Comment (GarageInc) [inline src/components/Navbar/NavbarMore/NavbarMore.module.scss:45]: $border-radius variable

File: src/components/Navbar/SantimentProductsTooltip/Logo.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+const Logo = ({ className }) => (
+  <Link className={cx(styles.logo, className)} to='/'>
+    <svg width='32' height='32' viewBox='0 0 32 32' fill='none'>
+      <path fill='#fff' d='M32 16a16 16 0 11-32 0 16 16 0 0132 0z' />
+      <path
+        fill='#181B2B'
+        d='M8.23 16a1.7 1.7 0 01-1.71 1.69A1.7 1.7 0 014.8 16c0-.93.77-1.69 1.72-1.69.94 0 1.71.76 1.71 1.69zM25.48 17.69A1.7 1.7 0 0027.2 16a1.7 1.7 0 00-1.72-1.69A1.7 1.7 0 0023.77 16c0 .93.77 1.69 1.71 1.69zM19.43 9.61a6.5 6.5 0 00-4.4-.67c-1.38.32-2.57 1.26-2.8 2.84-.18 1.3.22 2.33.9 3.14a9.31 9.31 0 002.27 1.77c.81.5 1.48.9 1.97 1.38.46.44.7.9.7 1.5 0 .69-.2 1.04-.43 1.24-.24.22-.63.38-1.19.42a5.43 5.43 0 01-3.3-.97l-1.21 1.56a7.42 7.42 0 004.65 1.37 3.92 3.92 0 002.4-.94 3.48 3.48 0 001.07-2.68c0-1.26-.55-2.19-1.28-2.9-.7-.67-1.6-1.2-2.32-1.65a7.52 7.52 0 01-1.8-1.36c-.35-.42-.55-.9-.45-1.61.08-.59.5-1.02 1.28-1.2a4.5 4.5 0 013 .5l.94-1.74z'
+      />
+      <path
+        fill='#D2D6E7'
```
Comment (GarageInc) [inline src/components/Navbar/SantimentProductsTooltip/Logo.js:15]: Colors for darkmode

**Review by GarageInc — APPROVED** [review]
I leaved minor comments

File: public/index.html (hunk 1 above)
Comment (haritonasty) [reply public/index.html:50]: Looks pretty for me :) We have the same logo on SAN project page as well
image

**Pull request merged**